### PR TITLE
hotfix: update doc according to recent PR

### PR DESCRIPTION
**Pull request by Simperfit**
Looking at the docs, i've seen some Resource left

### CODE REVIEW

File: core/external-vocabularies.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ class Product
 
     /**
      * ...
-     * @Property(iri="http://schema.org/name")
+     * @ApiProperty(iri="http://schema.org/name")
```
Comment (teohhanhui) [inline core/external-vocabularies.md]: You forgot to change the use statement.

File: core/operations.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ use ApiPlatform\Core\Annotation\Resource;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
- * @Resource(collectionOperations={"get"={"method"="GET"}})
+ * @ApiResource(collectionOperations={"get"={"method"="GET"}})
```
Comment (teohhanhui) [inline core/operations.md]: Take note of the use statement above.

File: core/external-vocabularies.md (hunk 1 above)
Comment (Simperfit) [inline core/external-vocabularies.md]: done

File: core/operations.md (hunk 2 above)
Comment (Simperfit) [inline core/operations.md]: done

**Comment by dunglas** [comment]
Thank you @Simperfit

**Pull request merged**